packages/frontend-core/src/utils/download.js

```diff
@@ -34,3 +34,35 @@ export async function downloadStream(streamResponse) {
 
   URL.revokeObjectURL(blobUrl)
 }
+
+export async function downloadFile(url, body) {
+  const response = await fetch(url, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(body),
+  })
+
+  if (!response.ok) {
+    return false
+  } else {
+    const contentDisposition = response.headers.get("Content-Disposition")
+
+    const matches = /filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/.exec(
```

Comment:
Can we break out this regex into a named constant to signify what it is supposed to do?

I'm guessing this is trying to find the file name from the content disposition, but its not 100% obvious.